frontend/public/components/resource-quota.jsx

```diff
@@ -30,11 +124,77 @@ const Row = ({obj: rq}) => <div className="row co-resource-list__item">
   </div>
 </div>;
 
+const CapacityIcon = ({capacity}) => {
+  let capacityIconClass = 'pficon-unknown';
+  if (capacity === 0) {
+    capacityIconClass = 'fa-circle-thin co-m-resource-quota-empty';
+  } else if (capacity > 0 && capacity < .5) {
+    capacityIconClass = 'pficon-resources-almost-empty';
+  } else if (capacity >= .5 && capacity < 1){
+    capacityIconClass = 'pficon-resources-almost-full';
+  } else if (capacity === 1) {
+    capacityIconClass = 'pficon-resources-full';
+  } else if (capacity > 1) {
+    capacityIconClass = 'pficon-warning-triangle-o';
+  }
+  const classPrefix = capacityIconClass.startsWith('pficon') ? 'pficon' : 'fa';
```

Comment:
Let's just add `fa` or `pficon` to the strings above rather than trying to calculate a prefix. The icons are all known, and it's less error prone.

Reply:
I can't win :-) I initially had it that way, but thought you'd prefer the more concise logic :-)

Reply:
fixed